examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/styled.ts

```diff
@@ -24,7 +24,6 @@ justify-content: space-between;
 export const ButtonContainer = styled(Button).attrs({
   noBackground: true,
 })`
-    background-color: ${({ highlight }) => (highlight ? '#d3eefc' : 'transparent')};
     flex-direction: row;
```

Comment:
It might be needed, did you check all the uses for this button?

Reply:
I've added it but not using it anymore